front/app/containers/PasswordRecovery/index.tsx

```diff
@@ -203,7 +203,8 @@ class PasswordRecovery extends React.PureComponent<
               <FormLabel htmlFor="email" labelMessage={messages.emailLabel} />
               <StyledInput
                 id="email"
-                type="text"
+                type="email"
```

Comment:
I also changed the type to 'email' to match the approach in `front/app/components/SignUpIn/SignIn/PasswordSignin.tsx`

Reply:
Perfect, it could well be that changing to type email is a needed part of the solution.